src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -118,6 +118,26 @@
       + "provide fixed width filenames that can be ordered by simple lexicographic sorting.";
   public static final int FILENAME_OFFSET_ZERO_PAD_WIDTH_DEFAULT = 10;
 
+  public static final String HDFS_AUTHENTICATION_KERBEROS_CONFIG = "hdfs.authentication.kerberos";
+  private static final String HDFS_AUTHENTICATION_KERBEROS_DOC =
+      "Config indicating whether the HDFS cluster is using Kerberos to perform authentication. ";
+  private static boolean HDFS_AUTHENTICATION_KERBEROS_DEFAULT = false;
+
+  public static final String HDFS_CONNECTOR_PRINCIPAL_CONFIG = "hdfs.connector.principal";
+  private static final String HDFS_CONNECTOR_PRINCIPAL_DOC =
+      "The principal to use when the HDFS cluster is using Kerberos to perform authentication.";
+  public static final String HDFS_CONNECTOR_PRINCIPAL_DEFAULT = "";
+
+  public static final String HDFS_CONNECTOR_KEYTAB_CONFIG = "hdfs.connector.keytab";
+  private static final String HDFS_CONNECTOR_KEYTAB_DOC =
+      "The path to the keytab file for the HDFS connector principal. "
+      + "This keytab file should only be readable by the connector user";
+  public static final String HDFS_CONNECTOR_KEYTAB_DEFAULT = "";
+
+  public static final String KERBEROS_TICKET_RENEW_PERIOD_MS_CONFIG = "kerberos.ticket.renew.period.ms";
+  private static final String KERBEROS_TICKET_RENEW_PERIOD_MS_DOC =
+      "The period in milliseconds to renew the Kerberos ticket.";
+  public static final long KERBEROS_TICKET_RENEW_PERIOD_MS_DEFAULT = 60000 * 60;
```

Comment:
@gwenshap Yeah, apparently others will renew a lot more frequently, like on the order of minutes. I don't really get why people would want to do a lot of extra unnecessary work, but if they want to, I won't stop them :)